go.sum

```diff
@@ -310,6 +311,8 @@ github.com/tmc/grpc-websocket-proxy v0.0.0-20171017195756-830351dc03c6/go.mod h1
 github.com/ugorji/go v0.0.0-20180813092308-00b869d2f4a5/go.mod h1:hnLbHMwcvSihnDhEfx2/BzKp2xb0Y+ErdfYcrs9tkJQ=
 github.com/ulikunitz/xz v0.5.5 h1:pFrO0lVpTBXLpYw+pnLj6TbvHuyjXMfjGeCwSqCVwok=
 github.com/ulikunitz/xz v0.5.5/go.mod h1:2bypXElzHzzJZwzH67Y6wb67pO62Rzfn7BSiF4ABRW8=
+github.com/vbauzysvmware/go-vcloud-director/v2 v2.0.0-20190909085019-a4faa0b2c572 h1:dKauI+f7/xUe2Ky5O7cp9IiZep184vE3x9Cu1qz+zCU=
+github.com/vbauzysvmware/go-vcloud-director/v2 v2.0.0-20190909085019-a4faa0b2c572/go.mod h1:HonlGxbjJ1NAibWh99eE4/S2l6ZOZ5KJzKK1rh2a9vc=
```

Comment:
Reminder on personal references

Reply:
This will be removed after real tag happens